webapp/src/Controller/API/AbstractRestController.php

```diff
@@ -268,10 +268,7 @@ protected function listActionHelper(Request $request)
         $queryBuilder = $this->getQueryBuilder($request);
 
         if ($request->query->has('ids')) {
-            $ids = $request->query->get('ids', []);
-            if (!is_array($ids)) {
-                throw new BadRequestHttpException('\'ids\' should be an array of ID\'s to fetch');
```

Comment:
Why can this not happen anymore?

Reply:
->get with arrays is deprecated. Using ->all returns an array. Errors are handled by Symfony